ArduPlane/GCS_Mavlink.cpp

```diff
@@ -414,12 +414,12 @@ bool GCS_MAVLINK_Plane::try_send_message(enum ap_message id)
 #endif
         break;
 
-    case MSG_FENCE_STATUS:
 #if GEOFENCE_ENABLED == ENABLED
```

Comment:
Not sending this under the new code?

Reply:
This is handled by `GCS_MAVLink/GCS_Common.cpp:4549`